### DISCUSSION TITLE
I don't understand the usefulness of the Duplicates tab.

### DISCUSSION THREAD

**Discussion by etyrnal** [nextcloud/bookmarks#2120] [Q&A]
I don't understand the usefulness of the Duplicates tab.  

It doesn't seem to give the ability to prune, or select duplicates, or give us the ability to remove ALL of the duplicates leaving just a single bookmark our of each duplicate group.

What is the point of the Duplicates tab.  It's almost like the function is saying, "you have duplicates, but there's nothing you can do about it."  lol

**Comment by marcelklehr** [comment]
You can open the details of a bookmark listed under duplicates and see in which folders it resides, by clicking on one of those folders you can then remove it from there. I'm not aware of a feature that would simplify this without resorting to a strategy that would take away your choice which bookmarks to keep.
